File: Core/LedgerTransactions/LedgerTransactionDto.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace Core.LedgerTransactions
{
    public class LedgerTransactionDto
    {
        [Required]
        public int AccountId { get; set; }
        public int TransactionId { get; set; }

        public decimal Amount { get; set; }
        public DateTime DateTimeCreatedUTC { get; set; }

        public LedgerTransactionTypeEnum TransactionType { get; set; }
    }
}

File: AngularApp/src/app/swagger-proxy/models/ledger-transaction-dto.ts
/* tslint:disable */
import { LedgerTransactionTypeEnum } from './ledger-transaction-type-enum';
export interface LedgerTransactionDto {
  accountId: number;
  transactionId: number;
  amount: number;
  dateTimeCreatedUTC: string;
  transactionType: LedgerTransactionTypeEnum;
}
